dubbo-remoting/dubbo-remoting-zookeeper/src/main/java/org/apache/dubbo/remoting/zookeeper/support/AbstractZookeeperClient.java

```diff
@@ -41,13 +42,25 @@
 
     private volatile boolean closed = false;
 
+    private Set<URL> sourceURLs;
+
     public AbstractZookeeperClient(URL url) {
         this.url = url;

```

Comment:
Should we user 
this(url,new ArrayList<>());

Reply:
I have remove this method for refactoring the code.